x/avs/keeper/keeper.go

```diff
@@ -157,3 +171,25 @@ func (k Keeper) DeleteAVSInfo(ctx sdk.Context, addr string) error {
 	store.Delete(avsAddr)
 	return nil
 }
+
+// IterateAVSInfo iterate through avs
+func (k Keeper) IterateAVSInfo(ctx sdk.Context, fn func(index int64, avsInfo types.AVSInfo) (stop bool)) {
```

Comment:
Where did this function be called? Seemed no abci or endBlocker calling

Reply:
This provides a method for iterating the avs list, which needs to be traversed externally. Simply use it